.github/workflows/ci.yml

```diff
@@ -243,8 +243,7 @@ jobs:
         run: cargo test --locked --all-targets --features e2e,postgres -p fuel-indexer-tests -- --test-threads=1
 
   cargo-test-forc-index-plugins:
-    # if: needs.check-is-semver-branch.outputs.is_semver_branch != 'true'
-    if: false
+    if: needs.check-is-semver-branch.outputs.is_semver_branch != 'true'
```

Comment:
Looks like I forgot to turn the plugin test back on after the cyclic dependency broke the release in the past